util/annotations/helpers_test.go

```diff
@@ -151,3 +151,81 @@ func TestAddAnnotations(t *testing.T) {
 		})
 	}
 }
+
+func TestHasTruthyAnnotation(t *testing.T) {
+	tests := []struct {
+		name          string
+		obj           metav1.Object
+		annotationKey string
+		expected      bool
+	}{
+		{
+			name: "no val",
```

Comment:
Just to cover all cases, please add a test case to cover "annotation does not exist" => false

Reply:
done!